app/views/pages/_editor_liquid_help.html.erb

```diff
@@ -1,344 +1,76 @@
-    <p>
-      You'll need the question or answer's ID code, and the site. When using <code>{% stackoverflow %}</code> as the tag, the site will default to Stack Overflow.
-      For example:
-      <br>
-      <a href="https://stackoverflow.com/questions/24789130/colors-in-irb-rails-console">https://stackoverflow.com/questions/24789130/colors-in-irb-rails-console</a>
-      <ul>
-          <li>The question ID is: <code style="color: aquamarine; background-color: black;">24789130</code></li>
-      </ul>
-      <pre>{% stackoverflow <span style="color: aquamarine;">24789130</span> %}</pre>
-    </p>
-    <p>
-      For other Stack Exchange network sites, you'll need to provide the site's name and use <code>{% stackexchange %}</code> as the tag. For example:
-      <br>
-      <a href="https://diy.stackexchange.com/questions/169988/base-for-refrigerator-wine-shelf">https://diy.stackexchange.com/questions/169988/base-for-refrigerator-wine-shelf</a>
-      <ul>
-        <li>The question ID is: <code style="color: aquamarine; background-color: black;">169988</code></li>
-      <li>The site is <code style="color: orange; background-color: black;">diy</code></li>
-      </ul>
-      <pre>{% stackexchange <span style="color: aquamarine;">169988</span> <span style="color: orange;">diy</span> %}</pre>
-    </p>
-    <p>
-    For answers, you can get the answer's ID code by from the answer's "Share" link. For example:
-    <br>
-    <a href="https://diy.stackexchange.com/a/170185">https://diy.stackexchange.com/a/170185</a>
-    <ul>
-      <li>The answer ID is: <code style="color: aquamarine; background-color: black;">170185</code></li>
-      <li>The site is <code style="color: orange; background-color: black;">diy</code></li>
-    </ul>
-    <pre>{% stackexchange <span style="color: aquamarine;">170185</span> <span style="color: orange;">diy</span> %}</pre>
-    </p>
-    <h3>Wikipedia Embed</h3>
-    <p>Enter the full URL of the Wikipedia article you want to embed, with or without the anchor.</p>
-    <p>
-      <code>{% wikipedia https://en.wikipedia.org/wiki/Wikipedia %}</code>
-      <br />
-      <code>{% wikipedia https://en.wikipedia.org/wiki/Wikipedia#Diversity %}</code>
-    </p>
-    <h3>Asciinema Embed</h3>
-    <p>All you need is the Asciinema id:</p>
-    <code>{% asciinema 239367 %}</code>
-    <h3>Parsing Liquid Tags as a Code Example</h3>
-    <p>To parse Liquid tags as code, simply wrap it with a single backtick or triple backticks.</p>
-    <p><code>`{% mytag %}{{ site.SOMETHING }}{% endmytag %}`</code></p>
-    <p>One specific edge case is with using the <code>raw</code> tag. To properly escape it, use this format:
-    </p>
-    <p><code>`{% raw %}{{site.SOMETHING }} {% ``endraw`` %}`</code></p>
+    <% if @user_approved_liquid_tags.include? UserSubscriptionTag %>
+      <h4><%= community_name %> User Subscriptions</h4>
+      <p class="fs-s fw-bold">This liquid tag can only be created with articles by admins.</p>
```

Comment:
Quick, non-blocking suggestion (same as above!): let's update the "a" in "admins" to be capitalized and use "posts" in place of "articles" here. 😊 
```suggestion
      <p class="fs-s fw-bold">This liquid tag can only be created with posts by Admins.</p>
```